implementations/rust/ockam/ockam_entity/src/lease/json_proto.rs

```diff
@@ -1,14 +1,19 @@
+//! JSON Client for the Ockam Cloud InfluxDB TokenLeaseManager service. The client supports
+//! requesting a lease with a given TTL, organization and bucket. The service implementation is
+//! limited to standalone InfluxDB instances, not Influx Cloud.
 use crate::{Lease, TTL};
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use tracing::error;
 
+//! Fine grained permissions for a resource such as "read" and "write" for a bucket.
 #[derive(Serialize, Deserialize, PartialEq, Debug)]
 pub struct LeaseProtocolPermissions {
-    action: String,
-    resource: HashMap<String, String>,
+    action: String, // read or write
```

Comment:
```suggestion
    /// read or write
    action: String,
```
or similar.

And on for the other `//` comments that are providing documentation (this doc does show up if you pass `--document-private-items` to rustdoc)